articles/tokens/set-access-token-format.md

```diff
@@ -0,0 +1,40 @@
+---
+description: How to set the Access Token format
+topics:
+  - tokens
+  - access-tokens
+contentType:
+  - how-to
+useCase:
+  - invoke-api
+---
+
+# Set the Access Token Format
+
+Auth0 generates Access Tokens in two formats: 
+* Opaque strings: When not using a custom API
+* [JSON Web Tokens (JWTs)](/jwt): When using a custom API
+
```

Comment:
I think we need a bit more here -- what determines custom vs. not custom API

Reply:
I agree. We are going to take another pass through these to add more information at the next pass.